content/blog/2018-02-09-rprofile-julia-stewart-lowndes.md

```diff
@@ -0,0 +1,87 @@
+---
+slug: rprofile-julia-stewart-lowndes
+title: ".rprofile: Julia Stewart Lowndes"
+date: 2018-02-09
+preface: "In this occasional series, we interview someone using a loosely defined set of interview questions for the purpose of demystifying the creative and development processes of R community members. This interview was conducted by Sean Kross and prepared by Kelly O'Briant as part of an rOpenSci unconf17 project."
+authors:
+  - name: Sean Kross
+    url: seankross.com
+  - name: Kelly O'Briant
+    url: https://kellobri.github.io/
+categories: blog
+topicid:
+tags:
+- R
+- community
+- interviews
+- rprofile
+---
+
+<img src="/img/blog-images/2018-02-09-rprofile-julia-stewart-lowndes/lowndes_profile.jpg" alt="Julia Stewart Lowndes" style="margin: 0px 20px; width: 250px;" align="left">
+_Dr. Julia Stewart Lowndes is the Science Program Lead for the Ocean Health Index and works at the National Center for Ecological Analysis and Synthesis. She and Sean Kross discussed how data science, open science, and community can help reproducibility in research._
+
+---
+
+<br>
+
+[This interview occurred at the 2017 rOpenSci unconference]
+
+SK: I’m Sean Kross, I’m the CTO of the [Johns Hopkins Data Science Lab](http://jhudatascience.org/). Today I’m interviewing Julia Stewart Lowndes. What is your current preferred job title?
```

Comment:
Maybe say "Julia, what is your current preferred ..." otherwise the transition sounds awkward